microprofilecsv.html

```diff
@@ -5512,13 +5514,15 @@
 	gd.Samples2 = Samples2;
 	for(let i = 0; i < Samples.length; ++i)
 	{
+		if (isNaN(Samples[i])) continue;
 		min = Math.min(min, Samples[i]);
 		max = Math.max(max, Samples[i]);
 	}
 	if(Samples2)
 	{
 		for(let i = 0; i < Samples2.length; ++i)
 		{
+			if (isNaN(Samples[i])) continue;
```

Comment:
This should probably be Samples2 here(copy paste mistake)

Reply:
Fixed(force pushed)